demos/python_demos/llm_text_generation/servable_stream/model.py

```diff
@@ -132,17 +132,17 @@ def serialize_completions(batch_size, result):
 
 class OvmsPythonModel:
     def initialize(self, kwargs: dict):
-        print("-------- Running initialize")
+        print("-------- Running initialize", flush=True)
         self.ov_model = OVModelForCausalLM.from_pretrained(
             MODEL_PATH,
             device="AUTO",
             ov_config=OV_CONFIG,
             config=AutoConfig.from_pretrained(MODEL_PATH, trust_remote_code=True))
-        print("-------- Model loaded")
+        print("-------- Model loaded", flush=True)
```

Comment:
Maybe we should create logger? Probably out of scope for this task

Reply:
I agree. It would be a separate feature that we could add along with some custom exception type that user could throw and error message would be returned in the server response etc.